### PR TITLE
docs: Add basic example written in Typescript

### PR DESCRIPTION
**Pull request by islasjuanp**
I've updated the example written in Typescript.  I think this doc might help other people to start using the library. 

Related to issue #759

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -57,6 +73,31 @@ app.post('/user', [
 });
 ```
 
+The validations can also be written in TypeScript in a similar way: 
+
+```typescript
+// ...rest of the initial code omitted for simplicity.
+import { check, validationResult } from 'express-validator/check';
```
Comment (fedeci) [inline docs/index.md:80]:
```suggestion
import { check, validationResult } from 'express-validator';
```

**Review by fedeci — COMMENTED** [review]
Hi @islasjuanp I don't think this should be closed as it may be useful for developers, but I would add something like a tabbed "bar" to select between TS and JS.

**Comment by gustavohenke** [comment]
> I would add something like a tabbed "bar" to select between TS and JS.

Yes! This would be optimal.
Looks like it's available on Docusaurus now: https://v2.docusaurus.io/docs/markdown-features/#multi-language-support-code-blocks

**Comment by fedeci** [comment]
Yes, if I am not wrong it is available also with `docusaurus@1.x`.

Should we update to `v2` or implement this using `v1`?

**Comment by gustavohenke** [comment]
indeed is available on v1 :)
https://v1.docusaurus.io/docs/en/doc-markdown#language-specific-code-tabs

A move to v2 looks like more evolved work, so it can stay for later.

**Pull request closed without merging**